internal/connector/internal/handlers/connectors.go

```diff
@@ -88,7 +99,10 @@ func (h ConnectorsHandler) Create(w http.ResponseWriter, r *http.Request) {
 			convResource.Owner = user.UserId()
 			convResource.OrganisationId = user.OrgId()
 
-			// validate create operation if connector is assigned to a namespace
+			// namespace id is a required field if unassigned connectors are not supported
+			if !h.connectorsConfig.ConnectorEnableUnassignedConnectors && (convResource.NamespaceId == nil || *convResource.NamespaceId == "") {
```

Comment:
Wonder why we don't set `namespace_id` as required in the OpenAPI definition
 I don't think we have a case where a user can create a connect without having to set the `namespace_id`